src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -432,8 +432,8 @@ typedef VOID (*RtcOnMessage)(UINT64, BOOL, PBYTE, UINT32);
  *
  * https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-ondatachannel
  */
-typedef struct __RtcDataChannel RtcDataChannel;
-typedef VOID (*RtcOnDataChannel)(UINT64, RtcDataChannel*);
+struct __RtcDataChannel;
```

Comment:
Why these changes? If we don't typedef then we need to use struct keyword in every place of usage

Reply:
Could you please describe which compilers don't do forward ref? Every one of the compilers should handle it

Reply:
https://stackoverflow.com/questions/8367646/redefinition-of-typedef
This doesn’t work for pure C.

Reply:
I believe it's about type re-declaration and not forward decl

Reply:
It is the forwarding typedef. We had same issue on C producer on the old gcc compiler before.